samples/grpc/shopping-analytics-service-java/pom.xml

```diff
@@ -21,7 +21,7 @@
         <scala.binary.version>2.13</scala.binary.version>
         <akka-persistence-r2dbc.version>1.0.0-M2</akka-persistence-r2dbc.version>
         <!-- FIXME final release version -->
-        <akka-projection.version>1.3.0-M2</akka-projection.version>
+        <akka-projection.version>1.3.0-M2-1-c7838fe8-20221006-0811-SNAPSHOT</akka-projection.version>
```

Comment:
Don't check this in, CI automatically builds against current and tests against and users won't be able to build